test/fluree/db/ledger/docs/query/advanced_query.clj

```diff
@@ -121,6 +121,24 @@
            (-> (map #(get-in % ["person/follows" "person/follows" "person/follows" :_id]) res) set)))))
 
 
+(deftest aggregate-binding
+  (testing "Aggregate function using two-tuple in where clause"
+    (let [query      {:select ["?e" "?maxNum"]
+                      :where  [["?e" "person/favNums" "?favNums"]
+                               ["?maxNum" "#(max ?favNums)"]]}
+          db         (basic/get-db test/ledger-chat)
+          res        (async/<!! (fdb/query-async db query))
+          maxNumVals (->> res
+                          (map second)
+                          (into #{}))]
+
+      ;; all two-tuple results should have the identical ?maxNum
```

Comment:
I think these comments would be more useful in the optional `msg` argument of the [`is` test macro](https://clojuredocs.org/clojure.test/is). That way if/when this test fails, we'll get a printed error message in the console/repl:

```clojure
(is (= 1 (count maxNumVals))
    "all two-tuple results should have the identical ?maxNum")
```